More descriptive names for the GPU tracks

ronaldfw opened the pull request:
There are three names of queues that we can encounter (on AMD): "gfx", "sdmaX", and "comp_X.Y.Z". After this change, these names are used for more descriptive track names ("Graphics queue", "Transfer queue", "Compute queue"). The original name passed from the tracepoint is still used in the name to disambiguate if, for instance, multiple compute queues are used.

pierricgimmig commented on OrbitGl/GpuTrack.cpp:
```diff
@@ -351,3 +351,20 @@ std::vector<std::shared_ptr<TimerChain>> GpuTrack::GetAllChains() {
 }
+
+//-----------------------------------------------------------------------------
+void GpuTrack::SetTimeline(std::string_view timeline) {
```
nit. I find "timeline" to be confusing to describe a label or a name.

beckerhe replied:
I kind of agree but it's also my first time looking at that code. And I don't have a good suggestion for a name, so I personally would "fix" that problem by avoiding that function all the way. I would define a free function that contains the absl::StrFormat-logic from below and maps from a gpu-track name to a gpu-track label. (Please feel free to ignore that.)

ronaldfw replied:
Yes, it is. "Timeline" is the Linux kernel name for this, for example, used in the tracepoint format. 

I will follow @beckerhe 's suggestion to map the timeline to the display name.

ronaldfw replied:
Done.

beckerhe commented on OrbitGl/TimeGraph.cpp:
```diff
@@ -708,6 +706,7 @@ std::shared_ptr<GpuTrack> TimeGraph::GetOrCreateGpuTrack(
   std::shared_ptr<GpuTrack> track = gpu_tracks_[timeline_hash];
   if (track == nullptr) {
     track = std::make_shared<GpuTrack>(this, string_manager_, timeline_hash);
+    track->SetTimeline(string_manager_->Get(timeline_hash).value_or(""));
```
When looking at other usages of `track->SetName` and `track->setLabel` in the code base, it seems like we assign the raw value to the name and the processed human-readable value to the label. Here both get the processed value. So I'm wondering, do we actually need the distinction?

ronaldfw replied:
Yes, you are right. I will use the "raw" name (which is the timeline name) for SetName and the descriptive name for SetLabel. I wasn't sure where that made a difference, but it's actually used when filtering tracks. So, the distinction is actually important. Thanks.

ronaldfw replied:
Except that we don't seem to filter GPU tracks, but that's something we might want to change?

ronaldfw replied:
Done for the purpose of this PR, I think.

ronaldfw commented:
Addressed comments. What do you think about adding the OrbitGl namespace? @beckerhe @pierricgimmig

beckerhe commented:
> Addressed comments. What do you think about adding the OrbitGl namespace? @beckerhe @pierricgimmig

Thanks a lot! I'm in favor of adding the `OrbitGl` namespace. I've done that so far also for other new code in OrbitQt and OrbitCore. All the new modules (OrbitLinuxTracing, OrbitSsh, ElfUtils) are namespaced anyway.

ronaldfw commented:
> > Addressed comments. What do you think about adding the OrbitGl namespace? @beckerhe @pierricgimmig
> 
> Thanks a lot! I'm in favor of adding the `OrbitGl` namespace. I've done that so far also for other new code in OrbitQt and OrbitCore. All the new modules (OrbitLinuxTracing, OrbitSsh, ElfUtils) are namespaced anyway.

Ok, done. I've only made the minimal change that the new free function is now in OrbitGl. I'll follow-up to expand the scope later.
